### ISSUE TITLE
Incorrect CSV time export

### ISSUE THREAD

**Issue by ian-r-rose** [CityOfLosAngeles/automated-walk-bike-counter#65]
I ran the algorithm on a three hour video this week, and the CSV count export showed over four hours of counts. Seems like there is a bug in computing the elapsed time? Any thoughts @vahedi25411?

**Comment by ian-r-rose** [comment]
Here is the sample output:
```
Time,pedestrian,cyclist
0:15:00,75,0
0:30:00,119,0
0:45:00,103,2
1:00:00,107,5
1:15:00,0,0
1:30:00,68,3
1:45:00,81,3
2:00:00,0,0
2:15:00,67,2
2:30:00,0,0
2:45:00,49,0
3:00:00,40,0
3:15:00,52,0
3:30:00,0,0
3:45:00,49,0
4:00:00,50,1
4:15:00,0,0
```

I am also puzzled by the time intervals with zero counts, especially since others have a few dozen counts. Seems like a pretty stark difference to me, but it's tough to check against the video itself since the timestamps don't match up.

**Comment by vahedi25411** [comment]
Can you share the video?

**Comment by ian-r-rose** [comment]
@vahedi25411 I'm sharing a 10 minute version of the above with you on Google Drive. I get the following CSV export at 1 minute intervals:
```
Time,pedestrian,cyclist
0:01:00,5,0
0:02:00,0,0
0:03:00,5,0
0:04:00,0,0
0:05:00,8,0
0:06:00,3,0
0:07:00,3,0
0:08:00,6,0
0:09:00,6,0
0:10:00,8,0
0:11:00,10,0
0:12:00,0,0
0:13:00,3,0
0:14:00,0,0
```

**Comment by vahedi25411** [comment]
@ian-r-rose I have worked on this video an there is something weird about it. When I am checking the framerate of the video from file properties, it is 30, but when checking the framerate from OpenCV it returns 14.90.
Is it an original video or a video with changed framerate?

**Comment by ian-r-rose** [comment]
Fixed by #66
